### ISSUE TITLE
Search bar on top right is too dim

### ISSUE THREAD

**Issue by Joshforde** [ccnz/Ushahidi_Web#87]
UI guys (i.e. me) missed this functionality  -  top tight grey on grey search button searches the text of reports.      This is powerful for locating info   e.g. 'autism' in the search tab 'location' finds nothing.    'Autism' in top right locates the appropriate service

**Comment by timClicks** [comment]
Josh - suggested solution?

**Comment by Joshforde** [comment]
This needs UI review but reposition that 2nd search bar above/below category bar to search the report headers for keywords but in such a way it doesn't confuse users with the _location_ search bar

**Comment by hayatoc** [comment]
Agreed - I didn't even know this existed and I've been on the site every day all week.
Also functionality - try searching "new world" - it only searches "new"
